client/src/plugins/usage-statistics/UsageStatistics.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import debug from 'debug';
+
+import { PureComponent } from 'react';
+
+import Flags, { ET_ENDPOINT, DISABLE_REMOTE_INTERACTION } from '../../util/Flags';
+import Metadata from '../../util/Metadata';
+
+import eventHandlers from './event-handlers';
+
+const log = debug('UsageStatistics');
+
+const PRIVACY_PREFERENCES_CONFIG_KEY = 'editor.privacyPreferences';
+const USAGE_STATISTICS_CONFIG_KEY = 'ENABLE_USAGE_STATISTICS';
+const EDITOR_ID_CONFIG_KEY = 'editor.id';
+
+const PRODUCT_MODELER = 'Camunda Modeler';
+const PRODUCT_EDITION_COMMUNITY = 'Community';
+
+const FETCH_METHOD = 'POST';
+const FETCH_HEADERS = { 'Accept': 'application/json', 'Content-Type': 'application/json' };
+
+// ET endpoint is set to Travis as an env variable, passed to client via WebPack DefinePlugin
+const DEFINED_ET_ENDPOINT = process.env.ET_ENDPOINT;
+
+// In order to plug a new event handler:
+//
+// 1) Create a new event handler class under event-handlers
+// 2) This class should extend BaseEventHandler
+// 3) This class should use 'addData' super method to add an additional data to the payload.
+// 4) This class should use 'flush' super method to send a payload to ET
```

Comment:
This is outdated.

Reply:
🐵

Reply:
Fixed via: https://github.com/camunda/camunda-modeler/pull/1840/commits/527ed4f137909d320fc647252463ccda3035e52c